src/resources/rmt_esp32.cc

```diff
@@ -153,6 +152,13 @@ PRIMITIVE(config_rx) {
   return process->program()->null_object();
 }
 
+PRIMITIVE(set_idle_threshold) {
+  ARGS(int, channel_num, uint16, threshold)
+  esp_err_t err = rmt_get_rx_idle_thresh(static_cast<rmt_channel_t>(channel_num), &threshold);
```

Comment:
It's called `set`, but it calls `get`.  Is this what you want?

Reply:
I want `set`, so I can dynamically change the threshold. I should have caught this when I made the second argument a pointer.